src/Octoshift/Mannequins.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using OctoshiftCLI.Models;
+
+namespace Octoshift
+{
+    public class Mannequins
+    {
+        private readonly Mannequin[] _mannequins;
+
+        public Mannequins(Mannequin[] mannequins)
```

Comment:
do we need this ctor? it's only used in tests

Reply:
removd. It shouldn't haver survived a refactor